rpcserver.go

```diff
@@ -717,10 +717,9 @@ func (r *rpcServer) ListUnspent(ctx context.Context,
 
 		// Now that we know we have a proper mapping to an address,
 		// we'll convert the regular outpoint to an lnrpc variant.
-		outpoint := &lnrpc.ChannelPoint{
-			FundingTxid: &lnrpc.ChannelPoint_FundingTxidStr{
-				FundingTxidStr: utxo.OutPoint.Hash.String(),
-			},
+		outpoint := &lnrpc.OutPoint{
+			TxidBytes:   utxo.OutPoint.Hash[:],
```

Comment:
This is something I noticed the other day - this hash is actually reversed. The `String()` method is the one that actually reverses it. Not that it should/shouldn't be changed, but just something to note.

Reply:
@wpaulino yep, that's how bitcoind txids are serialized

Reply:
I'm aware - have just seen users confused on slack about this.

Reply:
blame satoshi 🤷‍♂️not much we can do

Reply:
I guess as a convenience we should just reverse them for the user anyway? Can be left as a follow-up though.

Reply:
why? these _are_ the raw bytes. the purpose of providing them is so that the user doesn't have to remember to reverse them. if it was simply a matter of decoding hex, then we wouldn't bother adding it

Reply:
you should be able to take this output and copy it into a `chainhash.Hash`. If we provide them reversed, now the user has to reverse them again